script/foreman-debug

```diff
@@ -246,9 +246,18 @@ else
 fi
 printv "Determined $OS distribution"
 
+clean_temp() {
+  [ "$NOTAR" -eq 1 ] && exit 0
+  # prevent from removing anything user-defined
+  [[ "$DIR" != /var/tmp* ]] && exit 0
+  printv "Cleaning $DIR"
+  rm -rf "$DIR"
+  exit 0
```

Comment:
This will override the original exit code of the code, try:

```bash
#!/usr/bin/bash

function clean_temp () {
  echo "cleaning"
  exit 0
}
trap clean_temp EXIT SIGHUP SIGINT SIGTERM
exit 10
```

Why we are doing it?

Reply:
Oh good catch, it was a bug.